doc/api/esm.md

```diff
@@ -437,8 +437,8 @@ The `--experimental-json-modules` flag is needed for the module
 to work.
 
 ```bash
-node --experimental-modules --entry-type=module index.js # fails
-node --experimental-modules --entry-type=module --experimental-json-modules index.js # works
+node --experimental-modules index.js # fails
```

Comment:
```suggestion
node --experimental-modules index.mjs # fails
```

Reply:
yeah, I can change it [reluctantly] to 'mjs', although it also works with 'js', if package.json "type" is set to "module" ...